config/symbol_addrs.txt

```diff
@@ -398,6 +400,9 @@ CcharmMost__Fv = 0x160D98; // type:func
 
 reload_post_death = 0x160DA0; // type:func
 
+// P2/gs.c
+ClearFrameBuffers__Fv = 0x168AF8; // type:func
```

Comment:
`g_pgsCur` and `g_pwsCur` are actually part of `world.c`, not `gs.c` (confusing, I know)